packages/CopyInput/src/CopyInput.js

```diff
@@ -46,48 +47,65 @@ function CopyInput(props) {
 
   return (
     <sc.CopyInput data-pka-anchor="copy-input" {...moreProps}>
-      <Input ref={inputRef} defaultValue={defaultValue} {...extendedInputProps} />
-      <div ref={buttonRef}>
+      <Input ref={inputRef} defaultValue={value} isReadOnly={isReadOnly} {...extendedInputProps} />
+      <div ref={buttonRef} data-pka-anchor="copy-input.button">
         <Button.Icon
           kind="primary"
           onClick={handleButtonClick}
-          onMouseEnter={() => setIsHoverTooltipOpen(true)}
-          onMouseLeave={() => setIsHoverTooltipOpen(false)}
+          onMouseOver={() => setIsHoverTooltipOpen(true)}
+          onMouseOut={() => setIsHoverTooltipOpen(false)}
+          onFocus={() => setIsHoverTooltipOpen(true)}
+          onBlur={() => setIsHoverTooltipOpen(false)}
         >
           <CopyIcon />
         </Button.Icon>
       </div>
       <Popover
+        a11yText={isClickedTooltipOpen ? I18n.t("copyInput.clicked_tooltip") : I18n.t("copyInput.hover_tooltip")}
         align="bottom"
+        getPositioningElement={() => buttonRef.current}
         isDark
+        isEager
         isOpen={isHoverTooltipOpen && !isClickedTooltipOpen}
-        getPositioningElement={() => buttonRef.current}
       >
         <Popover.Content>
           <Popover.Card>{I18n.t("copyInput.hover_tooltip")}</Popover.Card>
         </Popover.Content>
         <Popover.Tip />
       </Popover>
-      <Popover align="bottom" isOpen={isClickedTooltipOpen} getPositioningElement={() => buttonRef.current}>
+      <Popover
+        align="bottom"
+        getPositioningElement={() => buttonRef.current}
+        isOpen={isClickedTooltipOpen}
+        shouldKeepFocus
+      >
         <Popover.Content>
           <Popover.Card>{I18n.t("copyInput.clicked_tooltip")}</Popover.Card>
         </Popover.Content>
         <Popover.Tip />
       </Popover>
+      {isClickedTooltipOpen && (
+        <Toast kind={Toast.types.kind.VISUALLY_HIDDEN} aria-hidden>
+          Copied
+        </Toast>
```

Comment:
My bad, I thought I checked this, but the `aria-hidden` is counter productive it seems.  Let's get rid of that and instead use `canAutoClose autoCloseDelay={1000}` to make it disappear from the DOM after it gets read by the screen reader.  

If this seems really weird, I can explain what's going on offline.

Reply:
Oh, and it needs the actual `I18n.t("copyInput.clicked_tooltip")` too.